Automattic/studio
# Studio: Improve site notifications when a site is created or import is completed

Closes https://github.com/Automattic/dotcom-forge/issues/8461

## Proposed Changes

This PR adds notification for the case when a new site is created (either with or without an import file)

## Testing Instructions

* Pull the changes from this branch
* Start the app with `STUDIO_IMPORT_EXPORT=true npm start`
* Click on `Add site` in the left sidebar
* Add an import file to be imported (you can use a file that you previously exported from Studio)
* Add a new site
* Once the site is created, confirm that you see the notification saying `Your new site is up and running`
* Confirm in the notification center that it is the only notification that you get 

**Test through Import/Export tab**

* Pull the changes from this branch
* Start the app with `STUDIO_IMPORT_EXPORT=true npm start`
* Navigate to the **Import/Export** tab
* Drag and drop an import file (you can use a file that you previously exported from Studio)
* Start the import
* Confirm that you see the `Import complete` notifications
* Confirm that this is the only notification that you see
## Pre-merge Checklist
